src/pages/catalog/oer/index.tsx

```diff
@@ -0,0 +1,244 @@
+import eordynamics from "../../../lib/images/tam212.png";
+import eorsolids from "../../../lib/images/tam251.png";
+import eornumerical from "../../../lib/images/cs357.png";
+import eorphysics1 from "../../../lib/images/EORphysics1.png";
+import eorphysics2 from "../../../lib/images/EORphysics2.png";
+import eorpython from "../../../lib/images/EORpython.png";
+import eorthermo from "../../../lib/images/EORthermo.png";
+import sigcse2023 from "../../../lib/images/sigcse2023.png";
+import cs233Image from "../../../lib/images/cs-233.png";
+import ece6353fraida from "../../../lib/images/ece6353fraida.png";
+
+interface SharedQuestionCardProps {
+  image: ImageProps["src"];
+  title: string;
+  href: string;
+  institution?: string;
+  ownerName?: string;
+  ownerEmail?: string;
+  github?: string;
+}
+
+const SharedQuestionCard: React.FC<SharedQuestionCardProps> = ({
+  image,
+  title,
+  href,
+  institution,
+  ownerName,
+  ownerEmail,
+  github,
+}) => {
+  return (
+    <article className="card border-secondary overflow-hidden">
+      <Link href={href} className="position-relative">
+        <Image
+          src={image}
+          alt={title}
+          style={{
+            objectFit: "contain",
+            width: "100%",
+            height: "100%",
+            aspectRatio: "5 / 3",
+          }}
+        />
+      </Link>
+      <div className="card-body">
+        <Link href={href}>
+          <h3 className="card-title h5">{title}</h3>
+        </Link>
+        {ownerName && (
+          <p className="mb-1">
+            <strong>Shared by:</strong> {ownerName}{" "}
+            {ownerEmail && <a href={`mailto:${ownerEmail}`}>{ownerEmail}</a>}
+          </p>
+        )}
+        {institution && (
+          <p className="mb-1">
+            <strong>Institution: </strong> {institution}
+          </p>
+        )}
+        {github && (
+          <p className="mb-1">
+            <strong>View source</strong> on <Link href={github}>Github</Link>
+          </p>
+        )}
+      </div>
+    </article>
+  );
+};
+
+export default function Courses() {
+  const [showRequestCourseModal, setShowRequestCourseModal] =
+    React.useState(false);
+  return (
+    <React.Fragment>
+      <Head>
+        <title>OER | PrairieLearn</title>
+      </Head>
+
+      <PageBanner
+        title="Open Educational Resources (OER)"
+        subtitle="A catalog of questions to help you get started with PrairieLearn"
+      />
+
+      <div className="container-fluid my-3">
+        <div className="container-md">
+          <div className="alert alert-primary mb-0">
+            <p>
+              You&apos;ll be asked to sign in to PrairieLearn to view our
+              question catalog. If you want to use any of these questions, you
+              will first need to request your PrairieLearn course space, which
+              is always free for instructors.
+            </p>
+            <div className="col-md-12 text-center">
```

Comment:
I don't think this class is doing anything?

```suggestion
            <div className="text-center">
```